http-client/src/main/java/com/facebook/airlift/http/client/thrift/ThriftResponse.java

```diff
@@ -0,0 +1,71 @@
+package com.facebook.airlift.http.client.thrift;
+
+import com.facebook.airlift.http.client.HeaderName;
+import com.google.common.collect.ImmutableListMultimap;
+import com.google.common.collect.ListMultimap;
+
+import javax.annotation.Nullable;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class ThriftResponse<T>
+{
+    private final int statusCode;
+    private final String statusMessage;
+    private final ListMultimap<HeaderName, String> headers;
```

Comment:
Is there an interface we can use?

Reply:
There is a broader plan to use something similar to `com.facebook.presto.server.smile.BaseResponse` as an interface. Specifically for `ListMultimap` reference, that looks like an interface already.